config/resque-pool.yml

```diff
@@ -1,5 +1,4 @@
 # I'm splitting this into two pools temporarily because these names are too long
 # Hopefully once all robots are on resque-pool, we can make the queue names shorter
 # see https://github.com/nevans/resque-pool/issues/176
-"dor_gisAssemblyWF_approve-data_default,dor_gisAssemblyWF_approve-metadata_default,dor_gisAssemblyWF_assign-placenames_default,dor_gisAssemblyWF_author-metadata_default,dor_gisAssemblyWF_extract-boundingbox_default,dor_gisAssemblyWF_extract-iso19139_default,dor_gisAssemblyWF_extract-thumbnail_default,dor_gisAssemblyWF_finish-data_default,dor_gisAssemblyWF_finish-gis-assembly-workflow_default,dor_gisAssemblyWF_finish-metadata_default,dor_gisAssemblyWF_generate-content-metadata_default,dor_gisAssemblyWF_generate-geo-metadata_default,dor_gisAssemblyWF_generate-mods_default,dor_gisAssemblyWF_load-geo-metadata_default,dor_gisAssemblyWF_normalize-data_default,gisAssemblyWF_default": 5
-"dor_gisAssemblyWF_package-data_default,dor_gisAssemblyWF_start-assembly-workflow_default,dor_gisAssemblyWF_start-delivery-workflow_default,dor_gisAssemblyWF_wrangle-data_default,dor_gisDeliveryWF_finish-gis-delivery-workflow_default,dor_gisDeliveryWF_load-geoserver_default,dor_gisDeliveryWF_load-geowebcache_default,dor_gisDeliveryWF_load-raster_default,dor_gisDeliveryWF_load-vector_default,dor_gisDeliveryWF_seed-geowebcache_default,dor_gisDeliveryWF_start-gis-discovery-workflow_default,dor_gisDiscoveryWF_export-opengeometadata_default,dor_gisDiscoveryWF_finish-gis-discovery-workflow_default,dor_gisDiscoveryWF_generate-geoblacklight_default,dor_gisDiscoveryWF_load-geoblacklight_default,gisDiscoveryWF_default,gisDeliveryWF_default": 5
+"gisDiscoveryWF_default,gisDeliveryWF_default": 5
```

Comment:
Does `gisAssemblyWF_default` not need to be in this list anymore?

Reply:
Thanks, I missed that.